<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the private '.service_locator.RWA3FG5' shared service.

return $this->privates['.service_locator.RWA3FG5'] = new \Symfony\Component\DependencyInjection\ServiceLocator(array('user' => function (): \App\Entity\User {
    return ($this->privates['.errored..service_locator.RWA3FG5.App\Entity\User'] ?? $this->load('getUserService.php'));
}, 'validationErrors' => function (): \Symfony\Component\Validator\ConstraintViolationListInterface {
    return ($this->privates['.errored..service_locator.RWA3FG5.Symfony\Component\Validator\ConstraintViolationListInterface'] ?? $this->load('getConstraintViolationListInterfaceService.php'));
}));
